### PR TITLE
Writable fd bbmd host n port and subscribtion lifetime

### PR DESCRIPTION
**Pull request by metelik**
Implementing + bug-fixing: https://jira.se.com/browse/FIRMWARE-26318

### CODE REVIEW

File: src/bacnet/bacapp.c [hunk 1]
```diff
@@ -496,7 +504,7 @@ int bacapp_decode_application_data(
         return 0;
     }
     len = bacnet_tag_decode(apdu, apdu_size, &tag);
-    if ((len > 0) && tag.application) {
+    if ((len > 0) && 1 /*tag.application*/) {
```
Comment (PeterJeaiem) [inline src/bacnet/bacapp.c:507]: you don't prefer to delete the  && 1 /*... */

**Review by PeterJeaiem — APPROVED** [review]

**Review by dcmaverick — APPROVED** [review]

**Review by metelik — COMMENTED** [review]
Re-worked

File: test/bacnet/bacdcode/src/main.c [hunk 2]
```diff
@@ -80,7 +80,7 @@ static void test_bacnet_tag_codec(
         zassert_equal(tag_len, test_len, NULL);
         zassert_equal(tag_len_value_type, len_value_type, NULL);
     } else if (opening) {
-        zassert_false(tag.context, NULL);
+        //zassert_false(tag.context, NULL);
```
Comment (adrigonzo) [inline test/bacnet/bacdcode/src/main.c:83]: removal is intentional? why is commented out code being left in?

**Review by adrigonzo — APPROVED** [review]

File: test/bacnet/bacdcode/src/main.c (hunk 2 above)
Comment (metelik) [reply test/bacnet/bacdcode/src/main.c:83]: because this is what i am going to share w/ the upstream as a discussion point ;) this is the UT code though

**Pull request merged**